ingresses/production/docs-jujucharms-com.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
   namespace: production
   annotations:
     kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      rewrite ^ https://jaas.ai/docs$request_uri? permanent;
```

Comment:
Nah bra, do it the way I did it - handle this domain in the `jaas.ai` ingress and delete this ingress.